client/src/Components/LoginPage/Login.tsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import { TouchableOpacity , Image, Alert} from 'react-native';
+import { observer } from 'mobx-react-lite';
+import { RootStackParamList } from '../../types/navigation';
+import {useNavigation , NavigationProp} from "@react-navigation/native";
+import axios from "axios";
+import CustomButton from '../../CustomComponents/CustomButton'
+import CustomView from '../../CustomComponents/CustomView'
+import CustomText from '../../CustomComponents/CustomText'
+import CustomInput from '../../CustomComponents/CustomInput'
+import getLoginStore from '../../stores/loginStore'
+import getThemeStore from '../../stores/themeStore';
+import { darkMode, lightMode } from '../colors/colors';
+import getAuthStore from '../../stores/authenticationStore';
+import { Dimensions } from 'react-native';
+import { setToken } from './utility/mmkvLogin';
+
+
+const googleImage = require('../../../../assets/google-symbol.png');
+
+const Login : React.FC = observer(() => {
+
+  const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+
+ 
+  const handleLogin = async (): Promise<void> => {
```

Comment:
handling all this logic should be in the mobx store, only renders in tsx files

Reply:
DONE ✅